generators/client/files-angular.js

```diff
@@ -260,7 +260,7 @@ const files = {
                 { file: 'account/social/_social-register.component.html', method: 'processHtml' },
                 { file: 'shared/social/_social.component.ts', method: 'processJs' },
                 { file: 'shared/social/_social.component.html', method: 'processHtml' },
-                'shared/social/_social.service.ts'
+                'core/social/_social.service.ts'
```

Comment:
if a different folder is preferred then can you move the files to the `angularCore` section added

Reply:
I would only move the file to the `angularCore` from the `angularShared` section, the other sections group files by features (social, i18n...). Does it makes sense to you?

Reply:
It might be confusing to have core module files inside shared section. Here we try to group by folder mostly I guess